compiler/control/OMROptions.cpp

```diff
@@ -4390,10 +4390,10 @@ OMR::Options::setCounts()
          {
          while (s[0] == ' ')
             ++s;
-         if (isdigit(s[0]))
+         if (OMR_ISDIGIT(s[0]))
             {
             count[i] = atoi(s);
-            while(isdigit(s[0]))
+            while(OMR_ISDIGIT(s[0]))
```

Comment:
While you're here, please add the missing space after `while`; see also lines 4421, 4444 and changes in other files.